### ISSUE TITLE
fasta headers annotations?

### ISSUE THREAD

**Issue by Thomieh73** [voutcn/megahit#54]
Hi,
I have run an assembly with megahit on a very sparse metagenome. it was fast and it giving me a similar assembly as with the clc genomics assembler. :-)

Now I am trying to understand the fasta headers from the contig file, so I can decide what to do with the 1.7 million contigs?
e.g
>k99_5 flag=0 multi=2.0435 len=213

I see my sequences have the flags, 0, 1 and 3. So what do these flags mean?

Then "multi", what is the relation of "multi" with the kmer counts and the coverage. I have not been able to find it.

**Comment by voutcn** [comment]
Hi Thomas,

`flag` is a tag to represent the *connectivity* of a contig in the assembly graph. `flag=1` means the contig is standalone, `flag=2` a looped path and `flag=0` for other contigs.

`multi` is roughly the average kmer coverage. But the figure is not precise and if you want to quantify the coverage of a contig, I would suggest you align the reads back to the contigs by short read aligners.

Basically, you could just ignore the header and put the contigs to the subsequent analysis.

**Comment by Thomieh73** [comment]
Thanks for the explanations. That is helpfull.
